com.unity.netcode.gameobjects/Runtime/Messaging/MessageQueue/MessageQueueContainer.cs

```diff
@@ -812,8 +812,7 @@ public MessageQueueContainer(NetworkManager networkManager, uint maxFrameHistory
             Initialize(maxFrameHistory);
         }
 
-
-#if UNITY_EDITOR || DEVELOPMENT_BUILD
+#if UNITY_INCLUDE_TESTS
```

Comment:
I'm still not sure how this macro works and I wouldn't advise to change it if not necessary.

Reply:
This is defined in the com.unity.netcode.runtimetests.asmdef file:
    "defineConstraints": [
        "UNITY_INCLUDE_TESTS"
    ],
According to @JesseOlmer it is the preferred define to use for this particular scenario.